### PR TITLE
Traffic history

### PR DESCRIPTION
**Pull request by Georgeto**
TODO:
- [ ] Write additional more detailed tests for the `traffic_history` function

### CODE REVIEW

File: pycroft/model/traffic.py [hunk 1]
```diff
@@ -222,5 +225,146 @@ class CurrentTrafficBalance(ModelBase):
...
+    result = select([agg_hist]).order_by(agg_hist.c.bucket).offset(1)
+
+    return result
+
+
+traffic_history_function = Function(
+    'traffic_history', ['arg_user_id int', 'arg_start timestamptz', 'arg_interval interval', 'arg_step interval'],
+    'TABLE ("timestamp" timestamptz, credit numeric, ingress numeric, egress numeric, balance numeric)',
+    str(
+        traffic_history_query().compile(
```
Comment (lukasjuhrich) [inline pycroft/model/traffic.py]: OK, I am just gonna trust this one.
ph'nglui mglw'nafh Cthulhu R'lyeh wgah'nagl fhtagn. Please don't let me down.

**Comment by lukasjuhrich** [comment]
Merging this before `develop-ng` to trigger an earlier build.

File: pycroft/model/traffic.py (hunk 1 above)
Comment (mueslo) [reply pycroft/model/traffic.py]: a hideous function like that requires a docstring though 😕

Comment (lukasjuhrich) [reply pycroft/model/traffic.py]: @mueslo no it doesn't. Be quiet.

**Pull request merged**